aiodocker/docker.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 import base64
+from _ssl import PROTOCOL_TLS
```

Comment:
We should use `ssl` instead `_ssl` as underscored modules are intended for private use by non-underscored modules. `ssl` also provides the definition of `PROTOCOL_TLS`, too.